docs/components/icons/Example.vue

```diff
@@ -0,0 +1,35 @@
+<script setup>
+  import * as icons from '@warp-ds/icons/vue';
+
+  const mappedIconsBySize = Object.keys(icons).reduce((acc, current) => {
+    const matches = current.match(/\d+/g);          
+      
+    if(!acc[ matches[0]]) {
+      acc[matches[0]] = {[current]: icons[current]}
+    } else {
+      acc[matches[0]] = {...acc[matches[0]], ...{[current]: icons[current]}}
+    }
+    return acc;
+      
+  }, {});
+  
+  const getIconName = (icon) => icon.replace(/Icon|\d+/g, '');
+
+</script>
+
+<template>
+  <section v-for="(size) in Object.keys(mappedIconsBySize)" class="s-bg-primary-subtle rounded-8 p-24 mb-24">
```

Comment:
It's a small thing but I wanted to mention it here as part of your question regarding theme switching. 
When we implement theme-switcher for icons, the `s-bg-primary-subtle` will be light green. If we used `s-bg-subtle` the background color difference won't be too big on theme change.
With s-bg-primary-subtle:
<img width="769" alt="Screenshot 2023-09-22 at 11 02 58" src="https://github.com/warp-ds/tech-docs/assets/41303231/333d6ff3-7224-4a14-8fe1-6494a0ed224c">
<img width="744" alt="Screenshot 2023-09-22 at 11 02 46" src="https://github.com/warp-ds/tech-docs/assets/41303231/48914a98-1cc2-489a-a0c5-25f85c8a88fc">

With s-bg-subtle
<img width="728" alt="Screenshot 2023-09-22 at 11 01 44" src="https://github.com/warp-ds/tech-docs/assets/41303231/d10fefab-5bb2-46f2-ad38-36bdd91b596e">
<img width="725" alt="Screenshot 2023-09-22 at 11 02 12" src="https://github.com/warp-ds/tech-docs/assets/41303231/a8c2775e-7e73-4ab0-8aa7-6cae2a39a097">

Reply:
Hm, maybe it is better to use non-semantic color here? as I can't see the need of changing the color here, right? I haven't thought about the details when I added the color 🤔

Reply:
I wouldn't mind it here as long as it's not too big of a change on theme switching, or at least that the class name is semantically correct (primary didn't seem like one 😄). I'd do whatever is easiest for you!